### PR TITLE
[deckhouse] Move some modules from FE to EE/CE

### PR DESCRIPTION
**Pull request by z9r5**
## Description

Moves the following modules from FE to EE:
- Keepalived
- Network-gateway

Moves the following modules to CE:
- extended-monitoring (enabled by default)
- namespace-configurator (enabled by default)
- okmeter	
- openvpn	
- secret-copier

## Changelog entries
```changes
section: keepalived
type: feature
summary: The module is available in the Deckhouse Enterprise Edition
---
section: network-gateway
type: feature
summary: The module is available in the Deckhouse Enterprise Edition
---
section: extended-monitoring
type: feature
summary: The module is available in the Deckhouse Community Edition and enabled by default.
---
section: namespace-configurator
type: feature
summary: The module is available in the Deckhouse Community Edition and enabled by default.
---
section: okmeter
type: feature
summary: The module is available in the Deckhouse Community Edition but requires Okmeter license.
---
section: openvpn
type: feature
summary: The module is available in the Deckhouse Community Edition (is in experimental state).
---
section: secret-copier
type: feature
summary: The module is available in the Deckhouse Community Edition and enabled by default.
```

### CODE REVIEW

File: docs/documentation/_data/i18n.yml [hunk 1]
```diff
@@ -108,9 +108,9 @@ features:
   experimental:
     en: This feature is actively developed. It might significantly change in the future.
     ru: Находится в процессе активного развития. Функциональность может существенно измениться.
-  fe:
-    en: This feature is available in Enterprise Edition only. It might significantly change in the future. Currently, it's not intended to be used without special assistance from Flant.
-    ru: Доступно только в Enterprise Edition. Функциональность может существенно измениться. В настоящий момент не рекомендуется для самостоятельного использования.
+  proprietaryOkmeter:
+    en: To use this feature you need to buy an <a href="https://okmeter.io">Okmeter</a> license.
+    ru: Для использования этого функционала необходимо приобретение лицензии <a href="https://okmeter.io">Okmeter</a>.
```
Comment (diafour) [inline docs/documentation/_data/i18n.yml:113]:
```suggestion
    en: This feature requires an <a href="https://okmeter.io">Okmeter</a> license.
    ru: Для этого компонента требуется лицензия <a href="https://okmeter.io">Okmeter</a>.
```

File: ee/modules/450-keepalived/hooks/keepalived_instance_handler_test.go [hunk 2]
```diff
@@ -7,7 +7,6 @@ package hooks
 
 import (
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```
Comment (diafour) [inline ee/modules/450-keepalived/hooks/keepalived_instance_handler_test.go:10]: Is it OK? I see `Expect` calls from gomega in this file.

File: tools/build_includes/modules-FE.yaml [hunk 3]
```diff
@@ -1,22 +1,4 @@
 # Code generated by tools/build.go; DO NOT EDIT.
-- add: /ee/fe/modules/340-extended-monitoring
-  to: /deckhouse/modules/340-extended-monitoring
-- add: /ee/fe/modules/340-monitoring-applications
```
Comment (diafour) [inline tools/build_includes/modules-FE.yaml:4]: Should this file be kept?

File: ee/modules/450-keepalived/hooks/keepalived_instance_handler_test.go (hunk 2 above)
Comment (z9r5) [reply ee/modules/450-keepalived/hooks/keepalived_instance_handler_test.go]: It is a bug. Fixed. Thanks

**Comment by z9r5** [comment]
Build [passed](https://github.com/deckhouse/deckhouse-test-1/runs/6212339602?check_suite_focus=true) with the changed CI

File: werf.yaml [hunk 4]
```diff
@@ -349,10 +345,9 @@ ansible:
   - raw: rm -rf /var/cache/apk/*
 
   setup:
-  - name: "Migrate ee/fe internal packages imports"
+  - name: "Migrate ee internal packages imports"
     shell: |
       find /deckhouse/modules/* -type f -exec sed -i 's/github.com\/deckhouse\/deckhouse\/ee\/modules/github.com\/deckhouse\/deckhouse\/modules/g' {} +
```
Comment (apolovov) [inline werf.yaml:350]:
```suggestion
    - find /deckhouse/modules/* -type f -name '*.go' -exec sed -E -i 's|github.com/deckhouse/deckhouse/ee(/fe)?/modules|github.com/deckhouse/deckhouse/modules|g' {} +```

**Review by RomanenkoDenys — DISMISSED** [review]

**Review by RomanenkoDenys — DISMISSED** [review]

**Review by RomanenkoDenys — DISMISSED** [review]

File: .github/scripts/js/ci.js [hunk 5]
```diff
@@ -628,7 +628,7 @@ module.exports.checkValidationLabels = async ({ github, context, core }) => {
  *   /e2e/use/cri/containerd
  *   /deploy/web/stage v1.3.2
  *   /deploy/alpha - to deploy all editions
- *   /deploy/alpha/ce,ee,fe
+ *   /deploy/alpha/ce,ee
```
Comment (nabokihms) [inline .github/scripts/js/ci.js:631]: We do not delete fe in this PR, so let's revert this change.

File: dhctl/README.md [hunk 6]
```diff
@@ -55,7 +55,7 @@ clusterDomain: "cluster.local"
 apiVersion: deckhouse.io/v1
 kind: InitConfiguration
 deckhouse:
-  imagesRepo: registry.deckhouse.io/fe
+  imagesRepo: registry.deckhouse.io/ee
```
Comment (nabokihms) [inline dhctl/README.md:58]: This one also should ve reverted.

**Review by nabokihms — CHANGES_REQUESTED** [review]

File: .github/scripts/js/ci.js (hunk 5 above)
Comment (z9r5) [reply .github/scripts/js/ci.js]: Reverted.

File: dhctl/README.md (hunk 6 above)
Comment (z9r5) [reply dhctl/README.md]: Reverted

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @nabokihms. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/gcp'...
